modules/ossm-applying-certificates-to-multi-cluster-topology.adoc

```diff
@@ -0,0 +1,114 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-applying-certificates-to-multi-cluster-topology_{context}"]
+= Applying certificates to a multi-cluster topology 
+
+This procedure explains how to apply security certificates to a multi-cluster topology.
+
+.Prerequisites
+
+* You have access to two {ocp-product-title} clusters with external load balancer support. 
+
+* You have installed the {SMProduct} 3.0 Operator to two clusters.
+
+* You have created the root CA certificate and intermediate CA certificates for each cluster or someone has made them available for you. 
+
+[NOTE]
+====
+In this procedure, `cluster1` is the East cluster and `cluster2` is the West cluster. 
```

Comment:
Should this be in  uppercase (`CLUSTER1` and `CLUSTER2`), since the only time we see these terms are in the environment variable names (`CTX_CLUSTER1`, `CTX_CLUSTER2`)?

Reply:
done